src/components/BoardComponent.vue

```diff
@@ -374,6 +380,72 @@ onUnmounted(() => {
 <template>
   <section class="layout">
     <div class="board-with-controls">
+      <div class="order-button-group">
+        <input
+          v-if="store.isDebug"
+          v-model.number="pieceToPlace"
+          type="number"
+        />
+        <template
+          v-if="
+            props.state.G.config.placePriorityAssignment?.beforeTurn &&
```

Comment:
need to fix this condition, looks like this on t1
<img width="359" alt="Screenshot 2024-03-25 at 8 57 02 PM" src="https://github.com/cmavelis/zug/assets/16199008/315145db-ebd0-4656-b98a-3ca89e5a2412">